mongo/mongo.go

```diff
@@ -638,10 +639,24 @@ func installPackage(pkg string, pacconfer config.PackagingConfigurer, pacman man
 
 func installMongod(operatingsystem string, numaCtl bool) error {
 	if featureflag.Enabled(feature.MongoDbSnap) {
+		snapName := ServiceName
+
+		// If we're installing a local snap, then update provide
+		// an absolute path. snap install will then do the Right Thing (TM).
+		files, err := ioutil.ReadDir("/var/lib/juju/snap")
+		if err == nil {
+			for _, fullFileName := range files {
```

Comment:
Iterating the dir is not the best approach here. We know what the snap file is as passed to the bootstrap CLI and sent through the bootstrap process

Reply:
Will look again at how to access the bootstrap args from here.